test/go-tests/test_backuprestore.go

```diff
@@ -145,6 +144,11 @@ func Test_BackupRestore(t *testing.T) {
 	err = os.MkdirAll("config-svc-backup", os.ModePerm)
 	require.Nil(t, err)
 
+	defer func() {
+		err = resetTestPath("../../../go-tests")
+		require.Nil(t, err)
+	}()
```

Comment:
Is it not possible to directly pass a utils function to defer instead of having to define one again only to call the utils function inside?